package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Why need this file?

Reply:
currently, I no need to have this file. I will remove it.
Last time I create it to use import / export module , need to put `  "type": "module" ` into package.json to let it work.
 But in current code,  I don't import/export code between files, so I should remove it.

Reply:
Yes. Please remove. once remove, please resolve this conversation

Reply:
I have removed it.